lib/archethic/contracts/interpreter.ex

```diff
@@ -697,6 +697,7 @@ defmodule Archethic.Contracts.Interpreter do
            {{:atom, "to"}, _to},
            {{:atom, "amount"}, _amount},
            {{:atom, "nft"}, _nft_address}
+           # ??
```

Comment:
I don't get what to do here? Please brief me about prewalk. {}

Reply:
You should add a parameter to accept nft_id in the interpreter parser